### PR TITLE
Build BER Azure initial commit

### PR DESCRIPTION
**Pull request by faiz-bloq**
### Contributing guide
https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md

### Changelog
*
*
*
*
*

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@faiz-bloq__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

**Comment by azurecla** [comment]
__@faiz-bloq__, Thanks for signing the contribution license agreement so quickly! Actual humans will now validate the agreement and then evaluate the PR.
<br />Thanks, AZPRBOT;

**Comment by marleyg** [comment]
Kay, can you approve this pull request please.

**Comment by faiz-bloq** [comment]
@marleyg Not sure if misspelling "improve" or trying to ping @singhkay ... I did go and improve the README.md to be more descriptive.

File: ber-on-ubuntu/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+![BER on Ubuntu](https://raw.githubusercontent.com/faiz-bloq/azure-quickstart-templates/ber-on-ubuntu/ber-on-ubuntu/images/bloq.png)
```
Comment (singhkay) [inline ber-on-ubuntu/README.md]: The links below need to point to the location in this repo

File: ber-on-ubuntu/azuredeploy.json [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+        "description": "Password for the Virtual Machine."
+      }
+    },
+    "dnsLabelPrefix": {
+       "type": "string",
+       "metadata": {
+          "description": "Hostname assigned to the Public IP, also used as the VM Name. Must be lowercase. It should match with the following regular expression: ^[a-z][a-z0-9-]{1,61}[a-z0-9]$ or it will raise an error."
+       }
+    },
+    "installMethod": {
```
Comment (singhkay) [inline ber-on-ubuntu/azuredeploy.json]: This could just be a variable since there is only one allowed value

**Comment by singhkay** [comment]
@faiz-bloq Left minor comments to address

**Comment by faiz-bloq** [comment]
Addressed comments, thanks for reviewing @singhkay

**Comment by singhkay** [comment]
@faiz-bloq Can you take a look at the CI failure?

**Comment by faiz-bloq** [comment]
@singhkay It seems related to the most recent change. CI is looking for a script in the Azure repository, however the code here has yet to be merged; Can CI be configured to pull from this branch?

**Comment by singhkay** [comment]
@faiz-bloq CI can auto replace to your fork if you have the correct link as it would be after the merge. It's passing now, merged.

**Pull request merged**